config.m4

```diff
@@ -11,10 +11,7 @@ if test "$PHP_PDO_4D" != "no"; then
 	
 
 	
-	ifdef([PHP_CHECK_PDO_INCLUDES],
-  [
-    PHP_CHECK_PDO_INCLUDES
```

Comment:
Does there need to be 2 [PHP_CHECK_PDO_INCLUDES]  ?

Currently:
ifdef(
        [PHP_CHECK_PDO_INCLUDES],
        [PHP_CHECK_PDO_INCLUDES],
        [
....
        ]
)

With 2 PHP_CHECK_PDO_INCLUDES the pdo_inc_path is not set correctly, and leads to the following error "Cannot find php_pdo_driver.h"

Removing one of the PHP_CHECK_PDO_INCLUDES sets the pdo_inc_path correctly and compilation succeeds.